docs/developers/setup-documentation.rst

```diff
@@ -18,13 +18,17 @@ Setting up
 This presumes that you're familiar with the command line, and are using OSX or another UNIX-like system.
```

Comment:
I think this presumption is valid, because this section of the docs is aimed at theme developers. But on the other hand, not all devs like the command line, and many prefer GUI tools.

Some possible command-line resources:
- http://www.davidbaumgold.com/tutorials/command-line/
- https://www.codecademy.com/learn/learn-the-command-line
- http://cli.learncodethehardway.org/book/